packages/docs/getting-started/running-a-validator.md

```diff
@@ -299,30 +308,42 @@ In order to be elected as a Validator, you will first need to register your grou
 Register your Validator Group:
 
 ```bash
+# On your local machine
```

Comment:
\# Note that because we did not authorize a Validator signer for our Validator Group account, we register the Validator Group with the account key